python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_checks.py

```diff
@@ -364,6 +366,27 @@ def test_materialize_asset_and_checks(
     }
 
 
+def test_materialize_asset_and_checks_with_python_check(
+    test_asset_checks_manifest: Dict[str, Any], dbt_commands: List[List[str]]
```

Comment:
giga nit but giving fixtures the prefix test_ feels bad

Reply:
also doesn't originate with this PR

Reply:
this is the case across all dbt tests currently- I'm gonna push back and say you're welcome to put up a rename pr

Reply:
ya agreed it doesn't need to be resolved here

Reply:
can I get a stamp